hugegraph-api/src/main/java/com/baidu/hugegraph/auth/ContextGremlinServer.java

```diff
@@ -40,27 +38,21 @@
 import com.baidu.hugegraph.auth.HugeGraphAuthProxy.ContextThreadPoolExecutor;
 import com.baidu.hugegraph.util.Log;
 
-import io.netty.channel.EventLoopGroup;
-import io.netty.channel.epoll.EpollEventLoopGroup;
-import io.netty.channel.nio.NioEventLoopGroup;
-
 /**
  * GremlinServer with custom ServerGremlinExecutor, which can pass Context
  */
 public class ContextGremlinServer extends GremlinServer {
 
     private static final Logger LOG = Log.logger(ContextGremlinServer.class);
 
-    @SuppressWarnings("deprecation")
     public ContextGremlinServer(final Settings settings) {
         /*
-         * TODO: use GremlinServer(Settings, ExecutorService)
          * which can be obtained from https://github.com/apache/tinkerpop/pull/813
          *
          * NOTE: should call GremlinServer::configureMetrics() but it's private
          * settings.optionalMetrics().ifPresent(GremlinServer::configureMetrics)
          */
-        super(create(settings));
+        super(settings, newGremlinExecutorService(settings));
```

Comment:
update more comments

Reply:
GremlinServer::configureMetrics() is still private

Reply:
will call configureMetrics() in super class